File: tournaments/differentSymbolsNaive/differentSymbolsNaive.cpp
int differentSymbolsNaive(std::string s) {

  int result = 0;

  for (int i = 0; i < 26; i++) {
    bool found = false;
    for (int j = 0; j < s.size(); j++) {
      if (s[j] == 'a' + i) {
        found = true;
        break;
      }
    }
    if (found) {
      result++;
    }
  }
  return result;
}

File: tournaments/differentSymbolsNaive/differentSymbolsNaive.pl
#!/usr/bin/perl
use strict;
use warnings FATAL => 'all';

sub differentSymbolsNaive {
    my ($s) = @_;

    my @store = split('', $s);
    my %hash;

    foreach my $letter (@store) {
        $hash{$letter} = 0;
    }

    return scalar(keys(%hash));
}
